witness/src/Dscp/Witness/Web/Logic.hs

```diff
@@ -234,3 +240,16 @@ checkFairCV =
         { fairCVCheckResults = results
         , fairCVFullyValid = all and results
         }
+
+checkFairCVPDF :: forall ctx m. WitnessWorkMode ctx m => ByteString -> m (FairCVCheckResult, FairCV)
+checkFairCVPDF pdf = do
+    let maybeFairCV = do
+            base64json    <- Pdf.project (Pdf.MaxSearchLength (Just 2048)) (Pdf.PDFBody pdf)
+            fairCVencoded <- toMaybe $ BS.decode base64json
```

Comment:
Minor: `BS.decode` is not very telling name for `Base??` :/
(same for `Scanner` module).

Also, `project` already seems to apply `LBS.decode`, is it fine?

Reply:
No, it isn't. Thanks for notifying, you probably save me some debug time.